<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMenu</class>
 <widget class="QMainWindow" name="GameMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameMenu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="runSimulation">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Simulation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="simulationComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>781</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
